tests/align_split/test_align_split.h

```diff
@@ -573,25 +483,17 @@ SEQAN_DEFINE_TEST(test_align_split_insertion_in_reference_gaps_banded)
 
     SEQAN_ASSERT_EQ(seqan::CharString("AGCATTTTAGATAAGATAG"), gapsHL);
     SEQAN_ASSERT_EQ(seqan::CharString("AGCATGTTAGATAAGATAG"), gapsVL);
-    SEQAN_ASSERT_EQ(clippedBeginPosition(gapsHL), 0);
-    SEQAN_ASSERT_EQ(clippedBeginPosition(gapsVL), 0);
-    SEQAN_ASSERT_EQ(clippedEndPosition(gapsHL), 19);
-    SEQAN_ASSERT_EQ(clippedEndPosition(gapsVL), 19);
-
     SEQAN_ASSERT_EQ(seqan::CharString("CTGTGCTAGTAGGCAGTCAGCGCCTT"), gapsHR);
     SEQAN_ASSERT_EQ(seqan::CharString("CTGTGCTAGTAGGCAGTCAGCGCCAT"), gapsVR);
-    SEQAN_ASSERT_EQ(clippedBeginPosition(gapsHR), 30);
-    SEQAN_ASSERT_EQ(clippedBeginPosition(gapsVR), 30);
-    SEQAN_ASSERT_EQ(clippedEndPosition(gapsHR), 56);
-    SEQAN_ASSERT_EQ(clippedEndPosition(gapsVR), 56);
```

Comment:
You removed many assertion, because probably they are wrong. Why did you not replace theme with the new (correct) values?